data-service/server/src/resolvers.js

```diff
@@ -0,0 +1,46 @@
+const queries = {
```

Comment:
Queries look good - I think I'm seeing double though 👓  are these the same as the ones in queries.js?